<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PileupController</class>
 <widget class="QWidget" name="PileupController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pileup Controller</string>
  </property>
  <widget class="QCustomPlot" name="histogramInput" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>461</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="histogramOutput" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="LogScaleCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>28</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> Y -LogScale</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelXAxisMin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Axis Min:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditXAxisMin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>8</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditXAxisMax">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>8</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelXAxisMax">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Axis Max:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameInput_1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; text-decoration: underline;&quot;&gt;Input #1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FileName #1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInputFile_1">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HistID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>to ch.::</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHistID_1">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFrom_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>53</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTo_1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>53</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Take data from ch.:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameOutput">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelOutput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>60</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; text-decoration: underline;&quot;&gt;Output&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ExpSpecFileName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditExpFileName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Norm. pileup from ch.:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNormFrom">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNormTo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>to ch.:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Norm. and Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveAndAddCont">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save and add as contam.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save (as Id:8300,norm 1e6)</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameInput_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; text-decoration: underline;&quot;&gt;Input #2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FileName #2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInputFile_2">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HistID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>to ch.::</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHistID_2">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFrom_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>53</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTo_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>53</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Take data from ch.:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonCalculate">
   <property name="geometry">
    <rect>
     <x>612</x>
     <y>280</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShowInputs">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Inputs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Axis Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Axis Max:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYAxisMin">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>28</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYAxisMax">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>28</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>PileupController</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
